### PR TITLE
Update copyright year for files recently modified

### PR DESCRIPTION
**Pull request by dataclouder**
This change is needed to satisfy the automated copyright check.
I've removed the check from Travis, but the script can be used for our own pre-committing checks.

Signed-off-by: Giuseppe Maxia <gmaxia@vmware.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@ TEST?=./...
 GOFMT_FILES?=$$(find . -name '*.go' | grep -v vendor)
 
 #default: fmt test testrace vet
-default: fmtcheck vet build copyright
+default: fmtcheck vet build
```
Comment (lvirbalas) [inline Makefile]: Why did you remove the check, considering that you recently enhanced the copyright check to be smarter and only check recently changed files?

Comment (dataclouder) [reply Makefile]: The check [failed unexpectedly](https://travis-ci.org/vmware/go-vcloud-director/builds/475189768) on a file that was not modified recently. I need to understand why. 
I think that my algorithm works on the assumption of a centralized source of truth, but fails on a truly distributed repository.
